elephant/spectral.py

```diff
@@ -523,6 +529,543 @@ def multitaper_psd(signal, n_segments=1, len_segment=None,
     return freqs, psd
 
 
+def multitaper_cross_spectrum(signals, fs=1., nw=4, num_tapers=None,
```

Comment:
Use consistent defaults across functions. The parameter is `float`, but other functions in this PR are using the `int` 1 and this uses `float` `1.`. I suggest the same style as `welch_psd` and this docstring, where the float number is passed as `1.0`. Also, the default values in the `def` statement should always match what is mentioned as default in the docstring.